build.sbt

```diff
@@ -137,10 +138,10 @@ lazy val common = (project in file("common"))
         cormorant, cormorantGeneric, scalaMock, scalacheckShapeless, diffx
       )
     ),
-    addCompilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1")
-    // addCompilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1"),
-    // unmanagedJars in Compile ++= Seq(new java.io.File("/tmp/aws-msk-iam-auth-2.2.0-all.jar")).classpath,
-    // unmanagedJars in Runtime ++= Seq(new java.io.File("/tmp/aws-msk-iam-auth-2.2.0-all.jar")).classpath   
+    // addCompilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1")
+    addCompilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1"),
+    unmanagedJars in Compile ++= Seq(new java.io.File("/tmp/amazon-keyspaces-helpers-1.0-SNAPSHOT-jar-with-dependencies.jar")).classpath,
+    unmanagedJars in Runtime ++= Seq(new java.io.File("/tmp/amazon-keyspaces-helpers-1.0-SNAPSHOT-jar-with-dependencies.jar")).classpath
```

Comment:
I got the keyspaces helper to work by adding the jar to the root of the code base in the `lib` folder, then in here
```sbt
Runtime / unmanagedBase := new java.io.File("./lib")
```
or we can add the jar to the subproject's `lib` folder, then sbt would load it automatically without needing to modify `build.sbt`, i.e. just put the helper jar into `hmda-platform/lib` folder